cmd/minishift/cmd/openshift/openshift.go

```diff
@@ -22,8 +22,8 @@ import (
 
 var OpenShiftCmd = &cobra.Command{
 	Use:   "openshift SUBCOMMAND [flags]",
-	Short: "Interact with an Openshift Cluster",
-	Long:  "Interact with an Openshift cluster, check available versions and modify master config resources",
+	Short: "Interacts with your local OpenShift cluster.",
+	Long:  "Interacts with your local OpenShift cluster. Use the sub-commands to check available OpenShift versions, access services, and modify configuration resources.",
```

Comment:
I think we need to reword this. `minishift openshift` does not stand alone. It is the actual sub-commands which do something.